.github/actions/run-dagger-pipeline/action.yml

```diff
@@ -95,6 +95,14 @@ runs:
     - name: Run airbyte-ci
       shell: bash
       run: |
+        export _EXPERIMENTAL_DAGGER_RUNNER_HOST="unix:///var/run/buildkit/buildkitd.sock"
+        DAGGER_CLI_COMMIT=${{ inputs.dagger_cli_commit }}
+        DAGGER_TMP_BINDIR="/tmp/dagger_${DAGGER_CLI_COMMIT}"
+        export _EXPERIMENTAL_DAGGER_CLI_BIN="$DAGGER_TMP_BINDIR/dagger"
+        if [ ! -f  "$_EXPERIMENTAL_DAGGER_CLI_BIN" ]; then
+          mkdir -p "$DAGGER_TMP_BINDIR"
+          curl "https://dl.dagger.io/dagger/main/${DAGGER_CLI_COMMIT}/dagger_${DAGGER_CLI_COMMIT}_$(uname -s | tr A-Z a-z)_$(uname -m | sed s/x86_64/amd64/).tar.gz" | tar xvz -C "$DAGGER_TMP_BINDIR"
+        fi
```

Comment:
This seems like a bit more than a revert... what's going on here?

Reply:
This what we had before to run a specific dagger version (relative to a pinned commit). It's not new: https://github.com/airbytehq/airbyte/pull/27833/files